### PR TITLE
Add argument to UndoRedo's "clear_history()" to not increase the version

### LINKED ISSUE #23791: Closing a scene tab makes another become "edited"
**Godot version:**
9eb4b6d

**Issue description:**
When closing a scene tab, the next tab that gain focus will be treated as "edited":
peek 2018-11-18 01-44

### PR DESCRIPTION
**Pull request by YeldhamDev**
Fixes #23791.

@reduz, could you see if used it in the right location inside the `EditorNode` file?

### CODE REVIEW

File: doc/classes/UndoRedo.xml [hunk 1]
```diff
@@ -102,6 +102,8 @@
 		<method name="clear_history">
 			<return type="void">
 			</return>
+			<argument index="0" name="increase_version" type="bool" default="true">
+			</argument>
 			<description>
 				Clear the undo/redo history and associated references.
```
Comment (akien-mga) [inline doc/classes/UndoRedo.xml]: Would be good to mention what `[code]increase_version[/code]` does in the description.

**Comment by akien-mga** [comment]
Looks good to me, though it might be worth reviewing all uses of `clear_history()` to see if other situations would require not to increase the version.

For example it might make sense for the Revert action: https://github.com/godotengine/godot/blob/2219ba256ac5f38f9b2273c961eb255d7b54b8e8/editor/editor_node.cpp#L2006

**Comment by YeldhamDev** [comment]
@akien-mga Made requested changes. If there's other instances where `false` should be passed, they could be merged later.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**